src/storage/src/hummock/compactor.rs

```diff
@@ -565,9 +547,17 @@ impl Compactor {
         };
 
         let get_id_time = Arc::new(AtomicU64::new(0));
+        let max_target_file_size = self.context.options.sstable_size_mb as usize * (1 << 20);
+        let cache_policy = if !self.context.is_share_buffer_compact
+            && (self.compact_task.target_file_size as usize) < max_target_file_size
+        {
+            CachePolicy::Fill
```

Comment:
For shared buffer compact, why do we not fill block cache after local compact?

Reply:
Because it is a bug and I explained why I removed it. We can add it to our code in another PR in a correct way.

Reply:
> Because it is a bug and I explained why I removed it in description. We can add it to our code in another PR in a correct way.

Good catch! It is very suble. Just curious, how do you spot this bug?